qml/pages/VenueList.qml

```diff
@@ -155,6 +155,7 @@ Page {
                     leftMargin: Theme.horizontalPageMargin
                     rightMargin: Theme.horizontalPageMargin
 
+                    topMargin: Theme.paddingSmall
```

Comment:
Imo that worsens the look of the Sailfish app a little bit.
Old:
![old](https://cloud.githubusercontent.com/assets/8768660/22994784/75303364-f3c8-11e6-8db4-377b35287b66.jpg)
![old2](https://cloud.githubusercontent.com/assets/8768660/22994778/70540eec-f3c8-11e6-8090-20d2eda3337b.jpg)

New:
![new](https://cloud.githubusercontent.com/assets/8768660/22994787/799fba50-f3c8-11e6-91a8-136b86a394bb.jpg)

We probably need a our own Theme API that we implement for both Sailfish and v-play (as opposed to only implementing the Sailfish API for v-play).

But for now, this is okay.